src/Settings.XamlStyler

```diff
@@ -19,7 +19,8 @@
         "*:*, *",
         "PageSource, PageIndex, Offset, Color, TargetName, Property, Value, StartPoint, EndPoint",
         "mc:Ignorable, d:IsDataSource, d:LayoutOverrides, d:IsStaticText",
-		"IsEnabled, x:Load, Load"
+		"IsEnabled, x:Load, Load",
+        "Value, Maximum, Minimum"
```

Comment:
This is one of those rare instances where the ordering of the binding actually matters (the order of binding is determined by the literal order the properties are defined in XAML). This change puts Value above Maximum and Minimum, otherwise if Value was higher than Maximum in the UI prior to the underlying viewmodel change, it would get reset to be the Maximum value and trigger a two-way binding, setting the default int for Value to be 2 or 0 instead of 1.